integration/image_list.sample.toml

```diff
@@ -0,0 +1,5 @@
+alpine = "docker.io/library/alpine"
+busybox = "docker.io/library/busybox"
```

Comment:
The sample toml example should also add the :latest tag to these 2 images.

Reply:
Done